extensions/podman/src/extension.ts

```diff
@@ -116,10 +116,33 @@ async function updateMachines(provider: extensionApi.Provider): Promise<void> {
     connectionsToCreate.map(async machineName => {
       // podman.sock link
       let socketPath;
-      if (isMac) {
-        socketPath = calcMacosSocketPath(machineName);
-      } else if (isWindows) {
-        socketPath = calcWinPipeName(machineName);
+      try {
+        if (isWindows) {
+          socketPath = await execPromise(getPodmanCli(), [
+            'machine',
+            'inspect',
+            '--format',
+            '{{.ConnectionInfo.PodmanPipe.Path}}',
+            machineName,
+          ]);
+        } else {
+          socketPath = await execPromise(getPodmanCli(), [
+            'machine',
+            'inspect',
+            '--format',
+            '{{.ConnectionInfo.PodmanSocket.Path}}',
+            machineName,
+          ]);
+        }
+      } catch (error) {
+        console.debug('Podman extension:', 'Failed to read socketPath from machine inspect');
```

Comment:
should we log the root cause ? The error ?

Reply:
I'm not sure about this. There will be nothing actionable from user. Also, it will always spam this error on Windows before Podman 4.5.0 is released. Also, the calling code doesn't really try any sort of error parsing, the expected error here is "command exit code was not equal to 0". It adds some value, but really minor.

Reply:
so maybe in the error message adds at the end ' defaulting to computed path' or something like that ?